{"ResponseBody":{"Message":"You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '1,1, 'test_01')\r\n,(0 2,1, 'test_02')\r\n,(0 3,1, 'test_03')\r\n,(0 4,1, 'test_04')\r\n' at line 2","InnerMessage":null,"StackTrace":"   at BottomhalfCore.DatabaseLayer.MySql.Code.Db.ConsicutiveBatchInset(String firstProcedure, Object parameters, String secondProcedure, List`1 secondQuery) in D:\\ws\\Server\\OnlineDataBuilderServer\\BottomhalfCore\\DatabaseLayer\\MySql\\Code\\Db.cs:line 669\r\n   at OnlineDataBuilder.Controllers.WeatherForecastController.Get() in D:\\ws\\Server\\OnlineDataBuilderServer\\OnlineDataBuilder\\Controllers\\WeatherForecastController.cs:line 106\r\n   at lambda_method(Closure , Object )\r\n   at Microsoft.Extensions.Internal.ObjectMethodExecutorAwaitable.Awaiter.GetResult()\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.AwaitableObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)\r\n   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)\r\n   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)\r\n   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)\r\n   at SchoolInMindServer.MiddlewareServices.RequestMiddleware.Invoke(HttpContext context, CurrentSession currentSession) in D:\\ws\\Server\\OnlineDataBuilderServer\\OnlineDataBuilder\\MiddlewareServices\\RequestMiddleware.cs:line 65\r\n   at SchoolInMindServer.MiddlewareServices.ExceptionHandlerMiddleware.Invoke(HttpContext context, ApplicationConfiguration applicationConfiguration) in D:\\ws\\Server\\OnlineDataBuilderServer\\OnlineDataBuilder\\MiddlewareServices\\ExceptionHandlerMiddleware.cs:line 28"},"HttpStatusCode":500,"HttpStatusMessage":"You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '1,1, 'test_01')\r\n,(0 2,1, 'test_02')\r\n,(0 3,1, 'test_03')\r\n,(0 4,1, 'test_04')\r\n' at line 2","AuthenticationToken":""}{"ResponseBody":{"Message":"Fatal error encountered during command execution.","InnerMessage":"Parameter '@id' must be defined.","StackTrace":"   at BottomhalfCore.DatabaseLayer.MySql.Code.Db.ConsicutiveBatchInset(String firstProcedure, Object parameters, String secondProcedure, List`1 secondQuery) in D:\\ws\\Server\\OnlineDataBuilderServer\\BottomhalfCore\\DatabaseLayer\\MySql\\Code\\Db.cs:line 669\r\n   at OnlineDataBuilder.Controllers.WeatherForecastController.Get() in D:\\ws\\Server\\OnlineDataBuilderServer\\OnlineDataBuilder\\Controllers\\WeatherForecastController.cs:line 112\r\n   at lambda_method(Closure , Object )\r\n   at Microsoft.Extensions.Internal.ObjectMethodExecutorAwaitable.Awaiter.GetResult()\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.AwaitableObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)\r\n   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)\r\n   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)\r\n   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)\r\n   at SchoolInMindServer.MiddlewareServices.RequestMiddleware.Invoke(HttpContext context, CurrentSession currentSession) in D:\\ws\\Server\\OnlineDataBuilderServer\\OnlineDataBuilder\\MiddlewareServices\\RequestMiddleware.cs:line 65\r\n   at SchoolInMindServer.MiddlewareServices.ExceptionHandlerMiddleware.Invoke(HttpContext context, ApplicationConfiguration applicationConfiguration) in D:\\ws\\Server\\OnlineDataBuilderServer\\OnlineDataBuilder\\MiddlewareServices\\ExceptionHandlerMiddleware.cs:line 28"},"HttpStatusCode":500,"HttpStatusMessage":"Fatal error encountered during command execution.","AuthenticationToken":""}